rust/agama-cli/src/auth.rs

```diff
@@ -109,6 +122,10 @@ impl Credentials for MissingCredentials {
 fn jwt_file() -> Option<PathBuf> {
     Some(home::home_dir()?.join(DEFAULT_JWT_FILE))
 }
+/// Path to agama-live token file
+fn agama_token_file() -> Option<PathBuf> {
+    Some(home::home_dir()?.join(DEFAULT_AGAMA_TOKEN_FILE))
```

Comment:
What about:

```rust
home::home_dir().map(|p| p.join(DEFAULT_AGAMA_TOKEN_FILE)
```

So you do not need to wrap the expresion (which contains a `?`) on a `Some`. But I guess it is just a matter of taste.